IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.m

```diff
@@ -499,6 +525,36 @@ - (UIActivityIndicatorView *)prepareLoadingIndicator:(UIView *)rootView
     return loadingIndicator;
 }
 
+- (void)handlePKeyAuthChallenge:(NSString *)challengeUrl
+{
```

Comment:
Discussion:
I am thinking about putting such a function to `MSIDPKeyAuthHandler`, although such a challenge is not caught the same way as other handlers? Thoughts? :)

Reply:
Yes, I think you should.

Reply:
done